doc/whats_new.rst

```diff
@@ -416,6 +416,10 @@ Miscellaneous
 - :class:`dummy.DummyClassifier` and :class:`dummy.DummyRegressor`
   now accept non-finite features. :issue:`8931` by :user:`Attractadore`.
 
+   - Added ability for :func:`model_selection.cross_val_predict` to handle multi-label
```

Comment:
indentation error

Reply:
Given its relation to other fixes we'll put in [0.19.1](https://github.com/scikit-learn/scikit-learn/milestones/0.19.1), I think we should include this in the bug-fix release.